Dockerfile

```diff
@@ -28,7 +36,10 @@ ENV RAILS_ENV=production
 ENV RAILS_LOG_TO_STDOUT=true
 ENV PORT=5000
 
-EXPOSE 5000
+# set to 1 to automatically create DB and run migrations
+ENV AUTOMIGRATE=0
```

Comment:
Do we actually need to bake this into the image? Seems like we could leave out the default since clients have to pass ` --env AUTOMIGRATE =1` to do anything useful. Also can we use an underscore to separate the two words (i.e. `AUTO_MIGRATE`)?

Reply:
removed

Reply:
also added instructions for use to `README.md`